src/lib/event_mapper/impl/general_aws_event_mapper.py

```diff
@@ -100,7 +111,7 @@ def create_table_row(self, values: list, style: str = None) -> dict:
             row["style"] = style
         return row
 
-    def to_message(self, monitored_env_name: str, event: dict) -> dict:
```

Comment:
We are now both assigning an event as a class field in the constructor, and accept an event in method arguments - maybe we should leave only one event in the class? Do we really need the event as the class field?